### ISSUE TITLE
Deep linking fails on Docker Deployment

### ISSUE THREAD

**Issue by rockliffelewis** [radiantearth/stac-browser#365]
If you're using the provided [Dockerfile](https://github.com/radiantearth/stac-browser/blob/main/Dockerfile) to create a container for deployment, and you send someone a link to a collection or item, it will throw a 404 for them even though the URL is valid.

This is because the nginx in the container is looking for a file, and then bailing out when it cannot find one, rather than forwarding to the Vue app and letting the Vue Router handle it.

If you add in a `try_files` directive into the config, then it looks for a file first, and then turns to the Vue app as a fallback which behaves more like you would expect

eg.
```
    location / {
        root   /usr/share/nginx/html;
        index  index.html index.htm;
        try_files $uri $uri/ /index.html;
    }
```

**Comment by m-mohr** [comment]
A PR would be welcome.

**Comment by m-mohr** [comment]
I think this will actually be solved by #349
